parlai/agents/seq2seq/seq2seq.py

```diff
@@ -158,6 +158,12 @@ def add_cmdline_args(argparser):
                            help='Keep replies in the history, or not.')
         agent.add_argument('-pt', '--person-tokens', type='bool', default=False,
                            help='use special tokens before each speaker')
+        agent.add_argument('-ede', '--eval-decode', type=str, default='greedy',
+                           choices=['greedy', 'beam'],
+                           help='Type of search for the best sequence during eval phase')
```

Comment:
let's actually remove this and just do greedy when beam-size is 1 and beam search when beram-size > 1